### PR TITLE
MRG: Move haxby example and show the report

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
* Puts the Haxby GML decoding example in the decoding folder
* Display the report in sphinx-gallery

### CODE REVIEW

**Comment by thomasbazeille** [comment]
Moving the Haxby example is already included in #2476 with some modifications relying on it.
I'd rather keep it there and add there you're supplementary changes if it's ok.

So i suggest closing this PR

**Comment by GaelVaroquaux** [comment]
Thanks, @thomasbazeille : I hadn't seen it in #2476 because that later PR is written, and I hadn't reviewed it in details.

**Comment by jeromedockes** [comment]
sorry I missed @thomasbazeille  's comment. should this be closed then?

**Comment by bthirion** [comment]
@thomasbazeille can you end #2476, even without all the features ? If yes, we close  that one.

File: doc/themes/nilearn/static/nature.css_t [hunk 1]
```diff
@@ -1702,6 +1702,10 @@ div.sg-report {
     transform: scale(.95);
 }
 
+div.body iframe {
```
Comment (thomasbazeille) [inline doc/themes/nilearn/static/nature.css_t:1705]: @GaelVaroquaux Should I mirror this in the PR superseding this one ?

Comment (thomasbazeille) [reply doc/themes/nilearn/static/nature.css_t:1705]: ping @GaelVaroquaux

**Comment by GaelVaroquaux** [comment]
> @GaelVaroquaux Should I mirror this in the PR superseding this one ?

Yes please!

Thanks a lot!

**Comment by thomasbazeille** [comment]
Superseded by #2476

**Pull request closed without merging**